platform/nrf52dk/platform-conf.h

```diff
@@ -132,9 +132,9 @@ typedef uint32_t uip_stats_t;
 /*
  * rtimer.h typedefs rtimer_clock_t as unsigned short. We need to define
  * RTIMER_CLOCK_LT to override this
- */
 typedef uint32_t rtimer_clock_t;
 #define RTIMER_CLOCK_LT(a, b)     ((int32_t)((a) - (b)) < 0)
+*/
```

Comment:
PR #1619 changed that.  It's `RTIMER_CLOCK_DIFF()`, now, that needs to be defined when you want to widen the rtimers.